### DISCUSSION TITLE
Chrome Bypass Certificate checks - method for passing in arbitrary flags to Chrome in Ghosts client?

### DISCUSSION THREAD

**Discussion by rbreesems** [cmu-sei/GHOSTS#131] [General]
I am looking for way to get Chrome browsing to bypass certificate checks. I can manually do this by passing the --ignore-certificate-errors on the command line, but in looking at the Ghosts Client source code I do not see a method for passing in arbitrary command line flags to Chrome.

Before I make a new branch that allows arbitrary parameters to the browser executables, is there some other way for passing in these parameters?

**Comment by sei-dupdyke** [comment]
Not currently, but I can add this pretty easily, and I created an issue to get this done (hopefully this week).

**Reply by rbreesems** [reply]
Thanks Dustin. I have temporarily compiled a new version of Ghosts client locally with that flag hardcoded, when your version comes out will switch to that.

**Reply by sei-dupdyke** [reply]
Already fixed in #132 👍

**Reply by rbreesems** [reply]
Wow! Will grab and test!

**Comment by rbreesems** [comment]
Thanks, grabbed it and tested, works as expected.  Nice!

**Reply by sei-dupdyke** [reply]
Excellent, good to hear sir. LMK if there is anything else I can expedite.

**Comment by rbreesems** [comment]
Will do. We are pushing a traffic generator role using Ghosts in our environment this release, keeping it simple for now - just email and Chrome Browsing to start. Will expand on using more Ghosts features next release and will definitely be in touch.
